dotCMS/src/main/java/com/dotcms/rest/MapToContentletPopulator.java

```diff
@@ -158,6 +153,25 @@ private void processMap(final Contentlet contentlet,
         }
     } // processMap.
 
+    /**
+     * if we fail to establish a language param then we need to supply a fallback
+     * @param contentlet
+     * @return
+     */
+    private long fallbackLanguage(Contentlet contentlet) {
```

Comment:
Nice method I think this should be useful as an util method, rather than private